test/e2e_node/restart_test.go

```diff
@@ -57,6 +64,23 @@ func waitForPods(f *framework.Framework, pod_count int, timeout time.Duration) (
 	return runningPods
 }
 
+type DockerDaemonOption struct {
+	Live_restore bool `json:"live-restore"`
+}
+
+func getDockerVersion() (string, error) {
```

Comment:
thoughts on having a shared helper with what is in kubelet container manager?

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/container_manager_linux.go#L800